sbndqm/dqmAnalysis/PMT/ICARUS/fcl/icarus_beam_timing_monitor.fcl

```diff
@@ -0,0 +1,54 @@
+#include "redis_connection.fcl"
+#include "icarus_online_decoder_defs.fcl"
+#include "icarus_pmt_analysis_defs.fcl"
+
+process_name: ICARUSBEAMTIMING
+
+physics: {
+  
+  producers: {
+
+    daqPMT: @local::decodePMTOnline
+    daqTrigger: @local::decodeTriggerOnline
+
+  }
+
+  analyzers: {
+
+   timingana: @local::timingAnalysis
+    
+  }
+
+  my_producer_modules: [ daqTrigger, daqPMT ]
+  trigger_paths: [ my_producer_modules ]
+  
+  ana: [ timingana ] 
+  end_paths: [ ana ]
+
+}
+
+services: {
+
+  TFileService: { fileName: "waveformtest.root"}
+
+  RedisConnection: @local::redis_connection
+
+  InitMetricManager: {
+    connections: {
+      fast: @local::redis_metric_fast
+      slow: @local::redis_metric_slow
+      archive: @local::redis_metric_archive
+    }
+  }
+
+} 
+
+
+services.RedisConnection.host: "icarus-db01.fnal.gov"
+
+source:
+{
+  module_type: RootInput
+  fileNames: ["/data1/sbndaq_dl01_r001248_sr01_20190702T192250_22_dl2.root"]
```

Comment:
Do we really want to leave an hard-coded input file?
Personally if it's a file that is always going to be available and is used as a test, I would rather leave this one unassigned (or assigned to `@nil`) and have a `test_icarus_beam_timing_monitor.fcl` like:
```
#include "icarus_beam_timing_monitor.fcl"

# test input:
source.fileNames: ["/data1/sbndaq_dl01_r001248_sr01_20190702T192250_22_dl2.root"]
```

Reply:
It's not needed. It's probably the last remnant of old tests that just kept being copied from fcl to fcl... Happy to change it to `@nil`.